package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "purescript-eth-core",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
We are actually at version  v3.0.0 and next one will be  v4.0.0, so please either revert this change and I will remove it later, or apply this change to it:

```
 {
   "name": "purescript-eth-core",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "private": true,
   "dependencies": {
     "bn.js": "^4.11.0",
     "keccak": "^1.0.2",
@@ -16,7 +14,5 @@
   },
   "scripts": {
     "test": "pulp test"
-  },
-  "author": "",
-  "license": "ISC"
+  }
 }
```

In PS libs we use package.json to only get npm deps and sometimes add `scripts` to it. versions of lib are managed using git tags https://github.com/f-o-a-m/purescript-eth-core/releases